Statistics/Correlation.hs

```diff
@@ -26,15 +29,20 @@ import Statistics.Test.Internal (rankUnsorted)
 -- | Pearson correlation for sample of pairs. Exactly same as
 -- 'Statistics.Sample.correlation'
 pearson :: (G.Vector v (Double, Double), G.Vector v Double)
-        => v (Double, Double) -> Double
-pearson = correlation
+        => v (Double, Double) -> (Double, PValue Double)
```

Comment:
you should probably document the return value?  that's not clear to an amateur like me at least.